pkg/app/provider/wopi/wopi.go

```diff
@@ -265,7 +259,7 @@ func (p *wopiProvider) GetAppURL(ctx context.Context, resource *provider.Resourc
 		}
 	}
 
-	log.Info().Msg(fmt.Sprintf("wopi: returning app URL %s", appFullURL))
+	log.Info().Msgf("wopi: returning app URL %s", appFullURL)
```

Comment:
Structured logging should be always prefered over `Msgf` or `Sprintf` usage. In this case you should add the url with `.Str("url", appFullURL)`

Reply:
Makes sense, thanks. Will have a look at a few others, the logging in this part of the software was/is pretty incomplete...